Drukhari faction

Orta15a opened the pull request:
Added Drukhari faction secondaries

Orta15a committed:
Drukhari faction

Zyllos commented on 9e/factions.json:
```diff
@@ -307,6 +307,52 @@
+					"objectives": [
+						{
+							"id": "EW.09.01.01",
+							"name": "Take them alive !",
```
Capitalize Them and Alive

Orta15a replied:
Done on latest commit

Zyllos commented on 9e/factions.json:
```diff
@@ -307,6 +307,52 @@
+					"objectives": [
+						{
+							"id": "EW.09.02.01",
+							"name": "Beasts for the arenas",
```
Capitalize Arenas

Orta15a replied:
Done on latest commit

Zyllos commented on 9e/factions.json:
```diff
@@ -307,6 +307,52 @@
+					"objectives": [
+						{
+							"id": "EW.09.03.01",
+							"name": "Fear and terror",
```
Capitalize Terror

Orta15a replied:
Done on latest commit

Zyllos commented on 9e/factions.json:
```diff
@@ -307,6 +307,52 @@
+					"objectives": [
+						{
+							"id": "EW.09.04.01",
+							"name": "Herd the prey",
```
Capitalize Prey

Orta15a replied:
Done on latest commit

Zyllos reviewed (commented):
I would update the capitalization of these objective names.

Orta15a committed:
Capitalize and FAQ update

- Correct titles with uppercase
- Change Take Them Alive first sentence in accordance to FAQ

Orta15a commented:
Added FAQ update on latest commit.

Orta15a committed:
AdMech Update

Adding Admech faction objectives

Orta15a commented:
Added Admech faction objectives
